src/hooks/useLogDownloader/index.ts

```diff
@@ -71,12 +72,31 @@ const useLogDownloader = (
         setFileSize(getFileSize() + progress);
       },
     })
-      .then((logs) => {
+      .then(({ result: logs, trimmedLines }) => {
         // Remove the last log line if it is empty
         if (logs[logs.length - 1] === "") {
           logs.pop();
         }
         setData(logs);
+        if (trimmedLines) {
+          sendEvent({
+            downloaded: getFileSize(),
+            duration: Date.now() - timeStart,
+            name: "Log Download Incomplete",
+            reason: "Log line size limit exceeded",
+          });
+          reportError({
+            message: "Log line size limit exceeded",
+            metadata: {
+              trimmedLines,
+              url,
+            },
+            name: "Log download incomplete",
+          }).warning();
+          dispatchToast.warning(LOG_LINE_TOO_LARGE_WARNING, true, {
+            title: "Log not fully downloaded",
```

Comment:
What do you think about adding `shouldTimeout: false`? I feel like this isn't a transient notice like e.g. a test starting that will be easily observed elsewhere on the page, so we should make it possible for users to digest the toast before it's gone forever.

Reply:
That makes sense done!